akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/BehaviorBuilder.scala

```diff
@@ -4,11 +4,10 @@
 
 package akka.actor.typed.javadsl
 
-import java.util.function.Supplier
-
 import scala.annotation.tailrec
 
-import akka.japi.function.{ Function => JFunction }
+import akka.japi.function.Function
```

Comment:
We should rename such, because it's easy that the import is removed accidentally and then it becomes scala.Function
We can still call it JFuntion